reinstall.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+DIVIDER="----------------------------------------------------------------------"
+clear
+echo "$DIVIDER"; echo "-- ATAT Web API developer reinstall"; echo "$DIVIDER"
+echo "This script is for developer convenience. It assumes:"
+echo " - \$ATAT_WEB_API_DIR ($ATAT_WEB_API_DIR) is defined"
+echo " - nvm is installed and \$NVM_DIR ($NVM_DIR) is defined"
+echo "This will do the following:"
+echo " - operate from the currently selected git branch and state"
+echo " - remove ALL installed node versions under nvm"
+echo " - install the preferred node version specified in .nvmrc"
+echo " - install the latest AWS CLI"
+echo " - install the latest AWS CDK Toolkit"
+echo " - install the latest TypeScript"
+echo " - install all package dependencies"
+echo " - bootstrap, build, and test"
+echo " - display software tool versions"
+read -rp "Press enter to continue or Ctrl+C to cancel"
+. $NVM_DIR/nvm.sh
+. $HOME/.profile
+. $HOME/.bashrc
+echo; echo "[Remove node versions]" && rm -rf $NVM_DIR/versions/node/ && mkdir $NVM_DIR/versions/node/
+echo; echo "[Install AWS CLI]" && sudo apt install awscli
```

Comment:
Th script might stop here because of `apt` command for mac users. I'm assuming not too many people are using macOS, so it'll work for the majority. I use Homebrew to install and manage my packages like awscli.

Reply:
Definitely good feedback.  I've only run it on Ubuntu.

Reply:
> I am not sure that we want to put these in the root of the repo. Maybe a `scripts/` folder?

That makes sense.

Reply:
I've replaced `reinstall.sh` with `install.sh`.  It does slightly less and stays away from those OS-specific commands and tasks like installing `awscli`.